docs/authentication/session-management.mdx

```diff
@@ -0,0 +1,896 @@
+
+Applications may utilize cookies, local storage or other strategies to associate session with the end-user’s application. Beyond Identity doesn’t provide an API/SDK for this purpose; it is up to you to decide how to handle the end-user’s session. One option is store the JWT received from Beyond Identity in the end user’s application. Another solution is to store the JWT received from Beyond Identity on the server and use a cookie. This is a common solution offered by many web frameworks and is more secure as the JWT is more sensitive.
+
+### Create a Token
+
+To request tokens for an app programmatically, send a request to the app's token request API.
+
+These requests follow the OAuth and OIDC protocols.
+
+#### Token Request URL and Endpoints
+
+The token request API URL is based on the Beyond Identity tenant, realm, and application for which a token is being requested.
+
+The API host name is prefixed with "auth-us" or "auth-eu", depending on the region of the tenant.
+
+:::note  
+Note that the host name for token requests is distinct from the Beyond Identity Management API, which uses "api-us" or "api-eu".  
+:::
+
+There are two main endpoints for requesting tokens:
+
+The /authorize endpoint:
+
+```http
+https://auth-{us|eu}.beyondidentity.com/v1/tenants/{tenant_id}/realms/{realm_id}/applications/{application_id}/authorize
+```
+
+and the /token endpoint:
+
+```http
+https://auth-{us|eu}.beyondidentity.com/v1/tenants/{tenant_id}/realms/{realm_id}/applications/{application_id}/token
+```
+
+where:
+
+- '{tenant_id}' is the [ID of the tenant](/docs/next/find-tenant-id) in which the app is configured
+
+- '{realm_id}' is the **Realm Id** of the realm in which the application is configured
+
+- '{application_id}' is the **Application ID** of the app
+
+#### Token Request Scenarios
+
+There are three main scenarios for requesting an access token:
+
+- Using the OAuth [client credentials grant type with a confidential client](#client-credentials-grant-type-confidential-client)
+
+- Using the OIDC [authorization code grant type with a public client (PKCE required)](#authorization-code-with-pkce-public-client)
+
+- Using the OIDC [authorization code grant type with a confidential client (PKCE recommended)](#authorization-code-with-pkce-confidential-client)
+
+#### Client credentials grant type (Confidential client)
+
+import AccessTokenCreateConfClientCreds from '../includes/_access_token_create_conf_client_creds.mdx';
+
+<AccessTokenCreateConfClientCreds />
+
+#### Authorization code with PKCE (Public client)
+
+import AccessTokenCreatePublicClientPkce from '../includes/_access_token_create_public_client_pkce.mdx';
+
+<AccessTokenCreatePublicClientPkce />
+
+#### Authorization code with PKCE (Confidential client)
+
+import AccessTokenCreateConfClientPkce from '../includes/_access_token_create_conf_client_pkce.mdx';
+
+<AccessTokenCreateConfClientPkce />
+
+#### Example: Create a Token for the Beyond Identity Management API
```

Comment:
What do you think about propagating the examples to the part in the doc where you are talking about them? When they are listed below as a group, for me, it's hard to follow what example goes with with content.

Reply:
I'm not quite sure what the difference would be, but they are no longer part of this document 🙂